File: Kotlin/ROUND_2_A/src/main/kotlin/FootballScoreStats.kt
import FootballData.allPlayed

class FootballScoreStats {
    fun teamTotal(teamName: String): Int {
        val played: List<Game> = allPlayed
        var total = 0
        for (game in played) {
            if (game.homeTeam.equals(teamName)) {
                total += game.homeTeamScore
            }
            if (game.awayTeam.equals(teamName)) {
                total += game.awayTeamScore
            }
        }
        return total
    }

}

File: csharp/ROUND_2/ROUND_2_A/src/FootballScoreStats.cs
namespace ROUND_2_A;

public class FootballScoreStats
{
    public int TeamTotal(string teamName)
    {
        int total = 0;
        Game[] played = FootballData.GetAllPlayed();
        foreach (var game in played)
        {
            if (game.HomeTeam.Equals(teamName))
            {
                total += game.HomeTeamScore;
            }
            if (game.AwayTeam.Equals(teamName))
            {
                total += game.AwayTeamScore;
            }
        }
        return total;
    }
}